src/main/java/com/amazon/dlic/auth/http/saml/Saml2SettingsProvider.java

```diff
@@ -212,11 +212,11 @@ private SingleLogoutService findSingleLogoutService(IDPSSODescriptor idpSsoDescr
     }
 
     private String buildAssertionConsumerEndpoint(String dashboardsRoot) {
-
+        // Changed to fix bug in 2.1 
```

Comment:
There is no need to have such comment in the source file, it belongs to the commit message.

Reply:
+1